features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/des/des_alt.c

```diff
@@ -342,66 +361,71 @@ static int mbedtls_des_docrypt(uint16_t keyopt, uint8_t key[3][MBEDTLS_DES_KEY_S
     *tdes_key ++ = val;
     val = nu_get32_be(key[2] + 4);
     *tdes_key ++ = val;
-    
+
```

Comment:
Asides: One should add `volatile` qualifiers to the pointers to the configuration registers written in `TDES_SetKey`, `TDES_SetInitVect` and `TDES_SetDMATransfer` in  `nuc472_crypto.c`. The above code does it correctly.

Reply:
@hanno-arm Thanks for pointing out the drawback in our BSP. I've notified my colleague and it would be fixed in next BSP release. Because currently these functions ( and similar) are not called, I wouldn't make modification to it.

Reply:
@hanno-arm I updated BSP crypto driver to fix `volatile` issue in https://github.com/ARMmbed/mbed-os/pull/4925/commits/5e6a65bb81d6c9edd1698e2cb538c9ce5dfc9cab.